### ISSUE TITLE
Integrate with SwitchyOmega?

### ISSUE THREAD

**Issue by myleshorton** [getlantern/lantern#4291]
A user referenced using this project, which looks like a variant of GoAgent. Interestingly, it looks like they add some sort of integration with SwitchyOmega. Not top priority, but @fffw it would be interesting to look at exactly what they're doing to integrate whenever there's a free moment. Or @percyalpha maybe you can tell?

https://github.com/XX-net/XX-Net

**Comment by skepts** [comment]
yeah sounds good. there are already guides on the Chinese Internet about how to adapt Lantern to SwitchOmega. 
BTW, i saw your post on the unused proxy-on.pac [https://github.com/getlantern/lantern/pull/1472](url)
But why when choosing "manage system proxy" will add one PAC setting to IE?

**Comment by fffw** [comment]
It doesn't have deep integration with SwitchyOmega. There's an recent updated [Chinese wiki](https://github.com/XX-net/XX-Net/wiki/安装和使用-SwitchyOmega) about manually integration though an .crx file and configuration is [embedded in code base](https://github.com/XX-net/XX-Net/tree/master/SwitchyOmega).

**Comment by fffw** [comment]
@CPCer The PR you mentioned is two years ago and no longer applies to current version. What "manage system proxy" does is to alter PAC settings.

**Comment by CPCer** [comment]
@fffw thx 
so it means the PAC i see now is dynamically updated?

**Comment by fffw** [comment]
@CPCer Yes. You can verify it by `curl` the pac url.
